### PR TITLE
Only 1 socket for address, group users conections and save logs for stats (address users)

### PR DESCRIPTION
**Pull request by thisjrodriguez**
Only 1 socket for address, group users conections and save logs for stats (address users)

### CODE REVIEW

**Comment by korzhyk** [comment]
Hi! Good job! Please can you reformat code for using intent of 2 spaces.

File: src/proxy-good-queue.js [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+        
+            //try parse json
+            try {
+                data = JSON.parse(message);
+            } catch (e) {
+                return log("can't parse message as JSON from miner:", message);
+            }
+			
+			//extract user
+			var id_user  = data.id_user;
```
Comment (korzhyk) [inline src/proxy-good-queue.js]: Avoid using a `var`

File: src/proxy-good-queue.js [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+                    //get mensaje
+                    const stratumMessage = addressConections[address].buffer.slice(0, newLineIndex);
+                    
+                    //remove line from buffer
+                    addressConections[address].buffer = addressConections[address].buffer.slice(newLineIndex + 1);
+          
+                    //ad 1 user
+
+                    
+                    log("[ADDRESS]["+addressConections[address].users+"] "+ address);
```
Comment (korzhyk) [inline src/proxy-good-queue.js]:
````js
log(`[ADDRESS][${addressConections[address].users}] ${address}`);
````

Comment (thisjrodriguez) [reply src/proxy-good-queue.js]: proxy-good-queue.js ignores that file, it works best with proxy.js

File: src/proxy-good-queue.js (hunk 1 above)
Comment (thisjrodriguez) [reply src/proxy-good-queue.js]: ¿data return a id_user?

Comment (korzhyk) [reply src/proxy-good-queue.js]: No, use `let` and `const` instead of `var`.

**Comment by cazala** [comment]
Hey @thisjrodriguez I was working on a big refactor (e5675135844ba6bad1bcf531f6a8efbcfabbb5cc ) and because of the conflics I didn't merge this PR but I included the one-connection-per-address logic.

**Pull request closed without merging**